lib/stormpath-sdk/id_site/error.rb

```diff
@@ -0,0 +1,42 @@
+module Stormpath
+  module IdSite
+    class Error < Stormpath::Error
```

Comment:
I see. But, does the Jwt token include the full status code Json? In that case, we could just use that and construct the error from there instead of hard coding it. Otherwise, your approach is fine.